tests/unit/tf/examples/test_usecase_transformers_next_item_prediction.py

```diff
@@ -34,5 +40,9 @@ def generate_date():
     tb.cells[4].source = tb.cells[4].source.replace(
         "read_csv('/workspace/data/train_set.csv'", "read_csv('/tmp/train_set.csv'"
     )
-    tb.cells[30].source = tb.cells[30].source.replace("epochs=5", "epochs=1")
-    tb.execute()
+    tb.cells[31].source = tb.cells[31].source.replace("epochs=5", "epochs=1")
+    tb.cells[37].source = tb.cells[37].source.replace("/workspace/ensemble", "/tmp/ensemble")
+    tb.execute_cell(list(range(0, 38)))
+
+    with run_triton_server("/tmp/ensemble", grpc_port=8001):
```

Comment:
I wonder if it is better to use something like in this test: https://github.com/NVIDIA-Merlin/Transformers4Rec/blob/main/tests/integration/notebooks/test_getting_started_session_based.py

please see lines 82-106. we can also add an assert may be to check out if the number of scores (elements in the response) returned from Triton are matching with the cardinality of the item_id or now.

Reply:
changes made